index.html

```diff
@@ -43,13 +48,50 @@ <h1 class="main-header">Build fast, responsive sites with ALPHA UI</h1>
           Functionality.
         </p>
         <div class="main-btn">
-          <a href="/documentation.html" class="btn btn-primary">
+          <a href="/components/avatar/avatar.html" class="btn btn-primary">
             Get Started
           </a>
           <a href="#" target="_blank" class="btn btn-secondary">GitHub</a>
         </div>
+        <div class="main-installation">
+          <div class="main-installation-header">Installation</div>
+          <div class="main-installation-container">
+            <div class="main-installation-left">
+              <h3 class="main-title">
+                In order to start using our component library, you need to include
+                the said lines in the head tag of your html page.
+              </h3>
+            </div>
+            <div class="main-installation-right">
+              <iframe
+                  src="https://carbon.now.sh/embed?bg=rgba%28250%2C252%2C255%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Falphauidev.netlify.app%2522%2520%252F%253E"
+                  scrolling="no"
+                  style="
+                  width: 100%;
+                  height: 20rem;
+                  border: 0;
+                  transform: scale(1);
+                  overflow: hidden;
+                  
+                "
+                sandbox="allow-scripts allow-same-origin"
+              >
```

Comment:
in the deploy preview it shows : <link rel="stylesheet" href="https://alphauidev.netlify.app" />
You can't access your CL in this way.
Import all the component css in one file say components.css and give the CDN link of that file.
it'll be like : <link rel="stylesheet" href="https://alphauidev.netlify.app/components.css" />

Reply:
Thanks I'll do that